File: Project_CS/StackSpeedTest/StackSpeedTest.cs
using System;
using System.Collections.Generic;

namespace Project_CS
{
    public class StackSpeedTest
    {
        public void Main(int max)
        {
            System.Diagnostics.Stopwatch sw = new System.Diagnostics.Stopwatch();
            sw.Start();

            Stack<int> stack = new Stack<int>();
            
            for (int i = 0; i < max; i++)
            {
                stack.Push(i);
            }

            Console.WriteLine("stack.Push({0}) done.", max);
            Console.WriteLine("stack.Contains({0}) ... {1}", 0, stack.Contains(0));

            while (stack.Count > 0)
            {
            /*
                int size = stack.Count;
                int peek = stack.Peek();
                int pop = stack.Pop();
                Console.WriteLine("size = {0}, peek = {1}, pop = {2}", size, peek, pop);
            */
                stack.Pop();
            }

            Console.WriteLine("stack.Pop({0}) done.", 0);
            Console.WriteLine("stack.Count ... {0}", stack.Count);

            sw.Stop();

            Console.WriteLine("Execute time : " + (sw.ElapsedMilliseconds).ToString() + " [ms]\n");
        }
    }
}

File: Project_Java_ArrayDeque/StackSpeedTest.java
import java.util.*;

public class StackSpeedTest {
    public void Main(int max) {
        long start = System.currentTimeMillis();

        ArrayDeque<Integer> stack = new ArrayDeque<>();

        for (int i = 0; i < max; i++) {
            stack.push(i);
        }

        System.out.println(String.format("stack.push(%d) done.", max));
        System.out.println(String.format("stack.contains(0) ... %s", Boolean.toString(stack.contains(0))));

        while (stack.isEmpty() == false) {
            /*
            int size = stack.size();
            int peek = stack.peek();
            int pop = stack.pop();
            System.out.println(formatter.format("size = %d, peek = %d, pop = %d\n", size, peek, pop));
            */
            stack.pop();
        }

        System.out.println(String.format("stack.pop(%d) done.", 0));
        System.out.println(String.format("stack.isEmpty()   ... %s", Boolean.toString(stack.isEmpty())));

        long end = System.currentTimeMillis();

        System.out.println("Execute time : " + (end - start)  + " [ms]\n");
    }
}
